src/main/java/com/minecolonies/coremod/lib/Literals.java

```diff
@@ -0,0 +1,17 @@
+package com.minecolonies.coremod.lib;
+
+/**
+ * Stores some literals used in many places of the code.
+ */
+public class Literals
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1118)